server/hotrod/src/main/java/org/infinispan/server/hotrod/HotRodServer.java

```diff
@@ -432,12 +460,14 @@ public CacheInfo getCacheInfo(AdvancedCache<byte[], byte[]> cache, HotRodHeader
                .noFlags().withFlags(LOCAL_NON_BLOCKING_GET);
          if (cache.getStatus() != ComponentStatus.RUNNING) {
             // stay on the safe side
-            return new CacheInfo(localNonBlocking, true, true);
+            return new CacheInfo(localNonBlocking, true, true, true);
```

Comment:
I wonder which parameter the comment above refers to...

Reply:
:) I was assuming it was all of them - since the optimizations only fire if the values are false.

Reply:
I don't think this check is needed, we can't get a reference to a cache before it's started and if it's stopping/stopped we'll throw an exception immediately.

Reply:
I have no idea personally, just how it was added by @rvansa. He would know more about why this check is present. But I would be up for removing, however I wouldn't want to do it in this PR.

Reply:
Why not? :smiley: